trie/trieStorageManager.go

```diff
@@ -301,7 +301,10 @@ func (tsm *trieStorageManager) Get(key []byte) ([]byte, error) {
 		return nil, ErrContextClosing
 	}
 
-	val, _ := tsm.mainStorer.Get(key)
+	val, err := tsm.mainStorer.Get(key)
+	if checkIfClosingError(err) {
```

Comment:
is this ok? Shouldn't have been `!checkIfClosingError(err)` ? Also, I would call this simply `isClosingError` 

Here and on L317, L329

Reply:
If there is a closing err, it should not continue to search in other dbs, but return the err. Changed the func name.